### PR TITLE
Ra config

### PR DESCRIPTION
**Pull request by maraino**
### Description

This PR allows the automatic configuration, without ca.json, of linked RAs.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,10 +16,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 ## [Unreleased]
+### Added
+- Added automatic configuration of Linked RAs.
+- Send provisioner configuration on Linked RAs.
 ### Changed
-- Certificates signed by an issuer using an RSA key will be signed using the same algorithm as the issuer certificate was signed with. The signature will no longer default to PKCS #1. For example, if the issuer certificate was signed using RSA-PSS with SHA-256, a new certificate will also be signed using RSA-PSS with SHA-256.
-- Support two latest versions of Go (1.18, 1.19)
-- Vadlidate revocation serial number (either base 10 or prefixed with an appropriate base)
+- Certificates signed by an issuer using an RSA key will be signed using the
+  same algorithm as the issuer certificate was signed with. The signature will
```
Comment (dopey) [inline CHANGELOG.md:24]:
```suggestion
same algorithm used to sign the issuer certificate.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,10 +16,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+  no longer default to PKCS #1. For example, if the issuer certificate was
+  signed using RSA-PSS with SHA-256, a new certificate will also be signed using
+  RSA-PSS with SHA-256.
+- Support two latest versions of Go (1.18, 1.19).
+- Validate revocation serial number (either base 10 or prefixed with an
+  appropriate base).
+- Sanitize TLS options.
+- Allow different representations of the serial number on X509 revocation.
```
Comment (dopey) [inline CHANGELOG.md:32]: This is a copy of one I already added above.

File: CHANGELOG.md (hunk 1 above)
Comment (maraino) [reply CHANGELOG.md]: Fixed in 3c88a9cc

File: CHANGELOG.md (hunk 2 above)
Comment (maraino) [reply CHANGELOG.md]: Removed in 3c88a9cc

**Review by dopey — APPROVED** [review]
lgtm

**Pull request merged**